File: src/java/src/main/java/org/elastos/hive/subscription/payment/PayOrderParams.java
package org.elastos.hive.subscription.payment;

import com.google.gson.annotations.SerializedName;

class PayOrderParams {
	@SerializedName("transaction_id")
	private String transactionId;

	public PayOrderParams(String transactionId) {
		this.transactionId = transactionId;
	}
}

File: src/domain/payment/payorderparams.ts
export class PayOrderParams {
//	@SerializedName("transaction_id")
	private transactionId;

	public setTransactionId(transactionId: string): PayOrderParams {
		return this.transactionId;
	}

    public getTransactionId(): string {
        return this.transactionId;
    }
}
